lib/gds_api/test_helpers/support_api.rb

```diff
@@ -80,22 +80,31 @@ def stub_support_api_anonymous_feedback_organisation_summary(slug, ordering = ni
       end
 
       def stub_support_api_organisations_list(response_body = nil)
-        response_body ||= [{
-          slug: "cabinet-office",
-          web_url: "https://www.gov.uk/government/organisations/cabinet-office",
-          title: "Cabinet Office",
-          acronym: "CO",
-          govuk_status: "live"
-        }]
+        response_body ||= [
+          {
+            slug: "cabinet-office",
+            web_url: "https://www.gov.uk/government/organisations/cabinet-office",
+            title: "Cabinet Office",
+            acronym: "CO",
+            govuk_status: "live"
+          },
+          {
+            slug: "gds",
+            web_url: "https://www.gov.uk/government/organisations/gds",
+            title: "Government Digital Service",
+            acronym: "GDS",
+            govuk_status: "live"
+          }
+        ]
 
         stub_http_request(:get, "#{SUPPORT_API_ENDPOINT}/organisations").
           to_return(status: 200, body: response_body.to_json)
       end
 
       def stub_support_api_organisation(slug = "cabinet-office", response_body = nil)
         response_body ||= {
-          slug: slug,
-          web_url: "https://www.gov.uk/government/organisations/#{slug}",
+          slug: "cabinet-office",
+          web_url: "https://www.gov.uk/government/organisations/cabinet-office",
```

Comment:
This doesn't seem right, I think we want to use the `slug` param if it's passed in.

Reply:
Put back slug. 10x!